app/controllers/events_controller.rb

```diff
@@ -3,7 +3,6 @@ class EventsController < ApplicationController
 	def create
 		@team = Team.find_by(number: params[:team])
 		@competition = Competition.find_by(name: params[:competition]);
-		@goal = Goal.find_by(name: params[:goal]);
 		if @competition == nil || @team == nil || @goal == nil
```

Comment:
A Ruby hack for nil checks is to simply test the variable as it will only return true if it is non-nil. e.g. `if !@competition || !@team`